src/Components/Alert/Components/Filters/QuickMultipleSelectAlertFilter.tsx

```diff
@@ -65,3 +60,30 @@ export const QuickMultipleSelectAlertFilter: FC<QuickMultipleSelectAlertFilterPr
     </>
   )
 }
+
+export const handleFieldsWithMultipleValues = ({
```

Comment:
extracted this to make sure I can use it from outside this context

Reply:
Nice 👍 Could even go into a file in `src/Components/Alert/Helpers` or `src/Components/Alert/Utils` 🤔